WoofInstall.cmake.in

```diff
@@ -6,14 +6,18 @@ set(Python3_EXECUTABLE "@Python3_EXECUTABLE@")
 set(WIN32 "@WIN32@")
 
 function(win_textfile SOURCE DESTINATION)
-    execute_process(COMMAND
-        "${Python3_EXECUTABLE}" "${PROJECT_SOURCE_DIR}/win-textfile"
-        "${SOURCE}" "${DESTINATION}")
+    if(Python3_EXECUTABLE)
```

Comment:
The CMake `file()` command does have a `NEWLINE_STYLE` parameter. If this gets us rid of the Python dependency, we should probably use it.
https://cmake.org/cmake/help/latest/command/file.html#writing

Reply:
This is probably only required for MINGW anyway. We shouldn't pull in a Python dependency for cross-compilation and on MSVC anyway - only for the sake of fixing line endings.

Reply:
I think I will remove these changes, this is for installation only and I am not sure if we will be doing packages on Windows XP :smile:

Reply:
Alright. However, if you have any ideas how to get rid oy Python for merely changing text line endings, I am all ears. We could probably include a local stripped-down copy of unix2dos in `win32/`:
https://sources.debian.org/src/dos2unix/7.4.2-2/unix2dos.c/

Reply:
Add unix2dos.c to toosrc, build it then use it in the post-install stage? As an alternative, we can write this code in cmake (it's possible but not pretty) or just remove fixing line endings entirely.

Reply:
It's possible to write line ending fixing code in CMake? How would that look?

Reply:
In the end, I can live with text files with LF line ending getting installed on Windows and I am sure others can too.

Reply:
> In the end, I can live with text files with LF line ending getting installed on Windows and I am sure others can too.

I agree. But maybe in another PR? This one already seems big.

Reply:
Yes, okay. So, let's just revert all changes to that part of the code for now.